File: rvi-analyzer-server/src/main/java/com/rvi/analyzer/rvianalyzerserver/entiy/DefaultConfiguration.java
package com.rvi.analyzer.rvianalyzerserver.entiy;

import lombok.Builder;
import org.springframework.data.mongodb.core.mapping.Document;
import org.springframework.data.mongodb.core.mapping.Field;

@Document
@Builder
public class DefaultConfiguration {
    @Field(name = "customer-name")
    private String customerName;
    @Field(name = "operator-id")
    private String operatorId;
    @Field(name = "batch-no")
    private String batchNo;
    @Field(name = "session-id")
    private String sessionId;
}

File: mobile_app/lib/domain/default_configuration.dart
class DefaultConfiguration {
  final String customerName;
  final String operatorId;
  final String batchNo;
  final String serialNo;
  final String sessionId;

  DefaultConfiguration(
      {required this.customerName,
      required this.operatorId,
      required this.batchNo,
      required this.serialNo,
      required this.sessionId});

  Map toJson() => {
        'customerName': customerName,
        'operatorId': operatorId,
        'batchNo': batchNo,
        'serialNo': serialNo,
        'sessionId': sessionId,
      };
}
